client/signup/steps/website-content/components.tsx

```diff
@@ -0,0 +1,31 @@
+import styled from '@emotion/styled';
+import FormLabel from 'calypso/components/forms/form-label';
+
+export const HorizontalGrid = styled.div`
+	display: flex;
+	justify-content: space-between;
+	width: 627px;
+`;
+
+export const Label = styled( FormLabel )`
+	font-style: normal;
+	font-weight: 600;
+	font-size: 13px;
+	line-height: 20px;
+	text-align: center;
+	color: #2c3338;
+	text-decoration-line: underline;
+	letter-spacing: -0.16px;
+	text-overflow: ellipsis;
+	max-width: 195px;
+	white-space: nowrap;
+	overflow: hidden;
+`;
+export const SubLabel = styled( Label )`
```

Comment:
This gives a TS error: `Exported variable 'Label' has or is using name 'Props' from external module "/Users/aneesh/Projects/Automattic/wp-calypso/client/components/forms/form-label/index" but cannot be named.`

Reply:
Should work now can you check again?